CMakeLists.txt

```diff
@@ -43,7 +44,7 @@ enable_testing()
 # https://github.com/boostorg/test/commit/d63379a9b3ad33d263d55c119681047c1d28547c
 if ((DEFINED Boost_VERSION_MACRO AND ${Boost_VERSION_MACRO} GREATER 106000) OR (${Boost_VERSION} GREATER 106000))
     set(BOOST_ARGS_SEPARATOR "--")
-endif()
+endif ()
```

Comment:
Why ?

Reply:
To follow the coding style. Do you prefer I revert this modification and create a pull request for this specific change?

Reply:
ok